Sources/MusicXML/Complex Types/Note.swift

```diff
@@ -142,8 +146,22 @@ extension Note: Codable {
         let ties = try container.decodeIfPresent([Tie].self, forKey: .tie).map(Ties.init)
 
         // Decode pitch / unpitched / rest
-        let pitchUnpitchedRestContainer = try decoder.singleValueContainer()
-        let pitchUnpitchedOrRest = try pitchUnpitchedRestContainer.decode(PitchUnpitchedOrRest.self)
+        let pitchUnpitchedOrRest: PitchUnpitchedOrRest
+        do {
```

Comment:
Can we try the `if container.contains(.pitch)` method that @DJBen has bestowed upon us?

Reply:
ah, sure!